model/stats/query_test.go

```diff
@@ -0,0 +1,523 @@
+package stats
+
+import (
+	"testing"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/testutil"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type statsQuerySuite struct {
+	suite.Suite
+}
+
+func TestStatsQuerySuite(t *testing.T) {
```

Comment:
I think this suite is probably a little too large: the test cases are very large, and it might be better to have some state on the suite object and then reset it in a setupTest fixture to make it easier to test, and more importantly debug fixtures if they change.

Reply:
Done.